manifest.yaml

```diff
@@ -1,9 +1,9 @@
 id: lnbits 
 title: "LNBits"
-version: 0.11.1
+version: 0.11.1.1
 release-notes: |
-  * Update upstream to 0.11.1 - including the new Node Manager! Full release notes available [here](https://github.com/lnbits/lnbits/releases/tag/0.11.1).
-  * Bug fixes and refactoring.
+  * Fix configuration bug.
```

Comment:
Since the previous version was deindexed, you can possibly leave these prior release notes in. Up to you.

Reply:
Updated to include release notes from 0.11.1